feature-libs/cart/base/core/facade/active-cart.service.spec.ts

```diff
@@ -541,7 +541,7 @@ describe('ActiveCartService', () => {
   });
 
   describe('requireLoadedCart', () => {
-    let cartState;
+    let cartState: any;
```

Comment:
Can a more specific typer be added here?

Reply:
I suppose it could, however I simply added this type to avoid repetitive 'implicit type error' in IDE